ui/app/models/pki/certificate/base.js

```diff
@@ -48,9 +47,8 @@ export default class PkiCertificateBaseModel extends Model {
   @attr('string') serialNumber;
 
   // Parsed from cert in serializer
-  @attr('number', { formatDate: true }) issueDate;
   @attr('number', { formatDate: true }) notValidAfter;
-  @attr('number', { formatDate: true }) notValidBefore;
+  @attr('number', { formatDate: true, label: 'Issue date' }) notValidBefore;
```

Comment:
I think we want this to still be labelled as Not Valid Before?

Reply:
I pulled this from the issuer view - but I agree. And it will hopefully help mitigate any future confusion!